### ISSUE TITLE
Problem with Bot not connecting

### ISSUE THREAD

**Issue by Michai901** [OfficeDev/microsoft-teams-apps-expertfinder#8]
So I followed the guide step by step but it come sto a pint after the deployment of the template (which took around 20 minutes) where everything seems to be provisioned but i cant get the bot to work.

When I try to manually acess the Bot App Service's domain it redirects me automatically to an error page.

Screenshots of the base ressource name, redirect url and bot test attached.
![chatwrong](https://user-images.githubusercontent.com/61748751/80813218-ed0d9880-8bc9-11ea-93c2-c832ed3dda94.PNG)
![ressourcegrup](https://user-images.githubusercontent.com/61748751/80813222-eda62f00-8bc9-11ea-8ae0-8770e541723c.PNG)
![weterleitung](https://user-images.githubusercontent.com/61748751/80813224-eda62f00-8bc9-11ea-9fed-0632846278cb.PNG)

**Comment by Michai901** [comment]
After I deployed the packed manifest.zip to teams it allows me to show my prfile but any other action fails and asks me to sign-in without anythig ever happening.
![teamsview](https://user-images.githubusercontent.com/61748751/80838915-5f966c80-8bfa-11ea-8ed4-a3d850e1feb0.PNG)

**Comment by abbodh** [comment]
Hi @Michai901 - can you please check if the App permission in app registration has SharePoint scope permission "AllSites.Read"?

**Comment by Michai901** [comment]
Yes i just checked both:  User.Read permission for Graph.API and AllSites.Read in Sharepoint API permissions were set

**Comment by eliezerbnu** [comment]
I have same problem.

This error on application:

[Information] Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler: Failed to validate the token.Microsoft.IdentityModel.Tokens.SecurityTokenSignatureKeyNotFoundException: IDX10501: Signature validation failed. Unable to match key:kid: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.Exceptions caught:'[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.token: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.at System.IdentityModel.Tokens.Jwt.JwtSecurityTokenHandler.ValidateSignature(String token, TokenValidationParameters validationParameters)at System.IdentityModel.Tokens.Jwt.JwtSecurityTokenHandler.ValidateToken(String token, TokenValidationParameters validationParameters, SecurityToken& validatedToken)at Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler.HandleAuthenticateAsync()
2020-05-26 15:12:13.827 +00:00 [Information] Microsoft.AspNetCore.Authentication.JwtBearer.JwtBearerHandler: Bearer was not authenticated. Failure message: IDX10501: Signature validation failed. Unable to match key:kid: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.Exceptions caught:'[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.token: '[PII is hidden. For more details, see https://aka.ms/IdentityModel/PII.]'.

**Comment by nurshari** [comment]
Hi.. I seems to be having same issue. I have checked both User.Read permission for Graph.API and AllSites.Read in Sharepoint API permissions and it was set correctly.

**Comment by zencell** [comment]
Also have the same problem.

**Comment by rashmi-dixit** [comment]
@Michai901 - Please check in bot settings if the Login URL entered is **https://login.microsoftonline.com**

**Comment by FHFTechTeam** [comment]
I give up :( after several installation and following the guide to the T, it just doesn't work :(
![image](https://user-images.githubusercontent.com/43945483/128164320-d4888f83-c5c6-492a-8b9a-fd18bce58652.png)
